src/apps/intel_mp/intel_mp.lua

```diff
@@ -729,7 +733,11 @@ function Intel:rss_enable ()
 end
 function Intel:rss_key ()
    for i=0,9,1 do
-      self.r.RSSRK[i](math.random(2^32))
+      if rss_hash_key then
+         self.r.RSSRK[i](rss_hash_key)
+      else
+         self.r.RSSRK[i](math.random(2^32))
```

Comment:
If we were to keep this, we should reduce this to one line (`rss_hash_key or math.random(2^32)`.  But I think instead we should make the RSS tests set SNABB_RANDOM_SEED.  WDYT?

Reply:
Yeah that sounds reasonable, I will try the change out.